### ISSUE TITLE
Build fails silently if python 3 in path

### ISSUE THREAD

**Issue by mukusz** [nexe/nexe#360]
This is a
 - [x] Bug Report
 - [ ] Feature Request
 - [ ] Other

<br/>

With sample code (plus verbose, plus python path):
```const nexe = require('nexe')

nexe.compile({
  input: './build/test-nexe.js',
  output: './dist/test-nexe.exe',
  patches: [
    async (compiler, next) => {
      await compiler.setFileContentsAsync(
        'lib/new-native-module.js',
        'module.exports = 42'
      )
      return next()
    }
  ],
  python: 'C:/python27',
  loglevel: 'verbose',
}).then(() => {
  console.log('success')
})
```
---

I have output:
> √ Using input file as the main module: ./build/test-nexe.js
> √ Downloading Node.js source from: https://nodejs.org/dist/v8.2.1/node-v8.2.1.tar.gz
> √ Source already downloaded
> √ Compiling result
> √ Scanning existing binary...
> √ Configuring node build:
> √ Compiling Node with arguments: nosign,release
> √ Node binary compiled
> - Writing result to fileevents.js:182
>       throw er; // Unhandled 'error' event
>       ^
> 
> Error: ENOENT: no such file or directory, open '...\.nexe\8.2.1\Release\node.exe'
> 

Digging deeper into the vcbuild.bat tools\msvs\find_python.cmd returns error 9009. 

In find_python at line 44 there's `%p%python.exe -V 2>&1 | findstr /R "^Python.2.*" > NUL`, it should be `"%p%python.exe" -V 2>&1 | findstr /R "^Python.2.*" > NUL`

In vcbuild at line 518 there's `if errorlevel 1 echo Could not find python2 & goto :exit`, it should be `if %errorlevel% gtr 0 echo Could not find python2 & goto :exit`

<br/><br/>

- Platform(OS/Version): Windows 10 x64
- Host Node Version: 8.2.1
- Target Node Version: 
- Nexe version: 2.0.0
- Python Version: 3.5.1 & 2.7.13

**Comment by calebboyd** [comment]
This looks like a quirk that was introduced in 8.2.0 But has been fixed in 8.3.0! (node)

https://github.com/nodejs/node/blob/v8.3.0/tools/msvs/find_python.cmd

I'll leave this open for reference.
Working on getting the kinks in the beta worked out. Thanks for the report!

**Comment by calebboyd** [comment]
I went ahead and fixed this here: 
https://github.com/nexe/nexe/blob/master/src/patches/build-fixes.ts
